virttest/qemu_vm.py

```diff
@@ -2142,10 +2152,14 @@ def create_serial_console(self):
         except IndexError:
             raise virt_vm.VMConfigMissingError(self.name, "serial")
         log_name = "serial-%s-%s.log" % (tmp_serial, self.name)
+        if 'ppc' in self.params.get('vm_arch_name', arch.ARCH) and 'tcp'in self.params.get('pro_backend'):
+            create_serial_cmd = "nc %s %s" % (self.params.get('pro_host'), self.params.get('pro_port'))
+        else:
+            create_serial_cmd = "nc -U %s" % self.get_serial_console_filename(tmp_serial)
```

Comment:
Hi Will-Do,
    Thanks for you reviewing.
    I've checked code by pep8 before I send this PR but there is no error prompt from 2155 to 2158.Can you double check ? thanks a lot.
Best Regards,
Miriam Deng